docs/windows.md

```diff
@@ -254,11 +254,26 @@ Set-NetFirewallProfile -Profile Domain,Public,Private -Enabled False
 Firstly, install wins, kubelet, kubeadm using script `PrepareNode.ps1` provided
 by kubernetes. The third component [`wins`](https://github.com/rancher/wins) is
 used to run kube-proxy and antrea-agent on Windows host inside the Windows
-container. The following command downloads and executes `PrepareNode.ps1`:
+container. Specify the Node IP, Kubernetes Version and container runtime while
+running the script. If you do not specify any container runtime, it will be containerd
+by default. (Note: Docker has been deprecated since k8s version 1.24). You can also
+specify whether you want to install OVS and kube-proxy on the node.
 
 ```powershell
-curl.exe -LO "https://github.com/kubernetes-sigs/sig-windows-tools/releases/download/v0.1.5/PrepareNode.ps1"
-.\PrepareNode.ps1 -KubernetesVersion v1.23.5
+# Example:
```

Comment:
No runtime change examples?

Reply:
Maybe add commands like `.\Prepare-Node.ps1 -KubernetesVersion v1.27.0 -NodeIP 192.168.1.10`  and `.\Prepare-Node.ps1 -KubernetesVersion v1.27.0 -NodeIP 192.168.1.10 -ContainerRuntime docker`  with explaination.